src/assets/js/soundmeter.js

```diff
@@ -5,22 +5,39 @@
 // instantaneous and time-decaying volumes available for inspection.
 // It also reports on the fraction of samples that were at or near
 // the top of the measurement range.
-function SoundMeter (context) {
+function SoundMeter(callback) {
 	var AudioContext = window.AudioContext || window.webkitAudioContext;
 	this.context = new AudioContext();
 	this.instant = 0.0;
-	this.script = this.context.createScriptProcessor (2048, 1, 1);
+	this.audioMeter = document.getElementById('audiometer');
+	this.start = null;
+	this.script = this.context.createScriptProcessor(2048, 1, 1);
 	this.stopped = true;
 	var self = this;
+
+	this.update = function (timestamp) {
+		if (!self.start) self.start = timestamp;
+		var progress = timestamp - self.start;
+		if (progress > 1000) {
+			self.audioMeter.value = self.instant.toFixed(2) * 5;
+			self.start = null;
+		}
+		window.requestAnimationFrame(self.update);
```

Comment:
Moving this to requestAnimationFrame gives the browser more control, for instance inactive tab it can just not execute this. As this is not an important part to keep it alive. This is just drawing